### ISSUE TITLE
Update required glib version to 2.63.1

### ISSUE THREAD

**Issue by changwoo** [ibus/ibus#2197]
For the glib GDBusServer authenticating issue https://github.com/ibus/ibus/issues/2137, please update the required glib version in configure:

https://github.com/ibus/ibus/blob/e3262f08b9e3efc57808700823b0622ec03a1b5f/configure.ac#L55

The fix has been applied to glib 2.62.3 (backported) and 2.63.1. So using glib 2.63.1 is a safe choice.

**Comment by fujiwarat** [comment]
You're right however the required version is the too latest and I hesitate in changing it for every user.
